click/_termui_impl.py

```diff
@@ -130,13 +130,13 @@ def eta(self):
 
     def format_eta(self):
         if self.eta_known:
-            t = self.eta + 1
+            t = int(self.eta)
             seconds = t % 60
-            t /= 60
+            t //= 60
```

Comment:
On python 2, division was integer, but with pyton 3, the result was floating point. Since the algorithm clearly relies on an assumption of integer division, making it explicit was the solution to fix the travis python3 failures.
Why this hasn't been discovered earlier is probably because this code path hasn't had any testing.. Which it now does, to a lesser extent.